test/Orleans.Transactions.Azure.Test/GoldenPathTests.cs

```diff
@@ -10,6 +10,7 @@ public class GoldenPathTests : GoldenPathTransactionTestRunner, IClassFixture<Te
         public GoldenPathTests(TestFixture fixture, ITestOutputHelper output)
             : base(fixture.GrainFactory, output)
         {
+            fixture.EnsurePreconditionsMet();
```

Comment:
When is this necessary?

Reply:
Logically: every time you use a fixture that inherits from `BaseTestClusterFixture` and hence this method exists.
Practically: when the specific fixture you are using in your test class depends on the environment and hence overrides the `BaseTestClusterFixture.CheckPreconditionsOrThrow()` method. Otherwise you'll get exceptions (not SkipException) when you actually try to use the client in your test method.